test/components/dragBoxLayerTests.ts

```diff
@@ -106,6 +106,26 @@ describe("Interactive Components", () => {
         svg.remove();
       });
 
+      it("does not call callbacks after destroy() is called", () => {
+        let group = new Plottable.Components.Group([dbl]).renderTo(svg);
```

Comment:
so that we can fake the mouse event somewhere (dbl.content() will be removed after destroy())